ballerina-shell/modules/shell-core/src/test/resources/testcases/evaluator/basics.quoted.json

```diff
@@ -59,5 +59,10 @@
     "description": "Member access of escaped name.",
     "code": "person2[\"first name\"]",
     "expr": "\"Tom\""
+  },
+  {
+    "description": "Use unicode with escape in name.",
+    "code": "string 'unicode_\\u{2324} = \"Jane doe\"; 'unicode_\\u{2324}",
+    "expr": "\"Jane doe\""
```

Comment:
Shall we add some of the other cases you tested as well?

Reply:
Added in 0ce501dc6d16f3464a59ae4f5636ac25167baa84